### ISSUE TITLE
Q2d

### ISSUE THREAD

**Issue by QuackDoctorW** [dilevin/computer-graphics-csc317#36]
Is it fair to ask if a camera transform matrix is the world to camera or camera to world?

Thanks!

**Comment by BobShoaun** [comment]
Yep relaying this question, i think its pretty important to specify if C is world to camera coords or the other way round.

**Comment by dilevin** [comment]
Camera to world
